queries/rbs/indents.scm

```diff
@@ -0,0 +1,4 @@
+[(class) (module) (interface)] @indent.begin
+
+"end" @indent.dedent
```

Comment:
Don't see a point for this. Tldr is `@indent.dedent` is for all child nodes of the nodes defined, while `@indent.end` is for the final child node of an `@indent.end` block